contrib/intel/jenkins/summary.py

```diff
@@ -530,27 +530,36 @@ def __init__(self, logger, log_dir, prov, mpi, file_name, stage_name):
         super().__init__(logger, log_dir, prov, file_name, stage_name)
 
         self.mpi = mpi
-        if self.mpi == 'impi':
-            self.run = '/mpiexec'
-        else:
-            self.run = '/mpirun'
+        self.run = 'mpiexec'
+    def read_file(self):
+        previous = ""
+        with open(self.file_path,'r') as log_file:
+            for line in log_file:
```

Comment:
is there a reason to leave the line with capital letters? All the other summary functions use lowercase lines. Can we do the same thing here?

Reply:
There are a couple of places with "errors" which are not a part of the result line. I will give this a try though.

Reply:
if it doesn't work for this file type thats okay, maybe just put a comment explaining that for us to remember why in the future